edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -562,73 +562,73 @@ Above all, the competitor limit must not be exceeded for any reason. Competitor
+
+- How to read a nametag, or otherwise determine the competitor's competing, judging, scrambling, and/or running groups.
+- What to do when the competitor's group is called.
+- How to compete.
+  - Telling the judge they are ready.
+  - Inspection time, including penalties.
+  - How to start the timer properly.
+    - Hand position, particularly using fingers to start the timer.
+    - Waiting for the green light on the timer.
+  - How to stop the timer properly.
+    - Flat hands; no "karate chops".
+  - How to sign for attempts.
     - Check that the time matches the timer and is written legibly. Times which are not written legibly will be interpreted in the worst way possible. Competitors can ask their judge to re-write times if they are not easily readable.
-  - How to call over a Delegate
-- How to judge
-  - Asking the competitor if they are ready and waiting for the competitor’s ready signal
-  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts
-    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds
-    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured
+  - How to call over a Delegate.
+- How to judge.
+  - Asking the competitor if they are ready and waiting for the competitor's ready signal.
+  - Judging inspection, including lifting the cube cover, operating the stopwatch, and making callouts.
+    - Emphasize that only "8 seconds" and "12 seconds" are called out, not 15 or 17 seconds.
+    - Stopping the stopwatch when the competitor starts their solve, so that the actual time of the inspection is measured.
   - What to look for when the competitor starts their solve, including:
     - Timer start hand position
     - Whether the competitor is touching the puzzle
-  - How to judge during the solve
-    - Pay attention. Don’t cube, use your phone, or engage in conversation
-    - Don’t distract the competitor during the solve. Sit still and be quiet
-    - Watch the competitor for any regulation violations
-    - Pay close attention to how the competitor stops their timer, and make sure they are using their hands with palms flat and facing down
-  - Finishing the solve
-    - The first step is to evaluate whether the cube is solved
-    - Do not touch the puzzle, use only your eyes
+  - How to judge during the solve.
+    - Pay attention. Don't cube, use your phone, or engage in conversation.
+    - Don't distract the competitor during the solve. Sit still and be quiet.
+    - Watch the competitor for any regulation violations.
+    - Pay close attention to how the competitor stops their timer, and make sure they are using their hands with palms flat and facing down.
+  - Finishing the solve.
+    - The first step is to evaluate whether the cube is solved.
+      - Do not touch the puzzle, use only your eyes.
     - How to evaluate the cube for +2s:
```

Comment:
Many of these "How to...." don't have the ":".  We should be consistent.  My understanding is you can use either, but it feels weird to swap between the two.

Reply:
Fixed a few that seemed off. I left : where it could be turned into a single sentence with : and commas.